asQ/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .paradiag import paradiag  # noqa: F401
 from .diag_preconditioner import DiagFFTPC  # noqa: F401
+import asQ.post  # noqa: F401
```

Comment:
I think that we just want it to be part of paradiag, because it is key to the core usage of paradiag.

Reply:
Ok, so no nesting on the module?

Reply:
See commit [1a5b86d](https://github.com/colinjcotter/asQ/pull/33/commits/1a5b86dd387338d2c9cd2736f4bdeb58eb404350)